hudi-tests-common/pom.xml

```diff
@@ -208,6 +208,10 @@
                     <groupId>io.netty</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
```

Comment:
This is redundant as L208-210 already excludes the transitive dependency of `io.netty`.